packages/timeline/src/ReactAxisLabelTimeline.ts

```diff
@@ -0,0 +1,229 @@
+import { HTMLWidget } from "@hpcc-js/common";
+import { Border2 } from "@hpcc-js/layout";
+import { IAxisGanttData } from "./ReactAxisGantt";
+import { ReactTimelineSeries } from "./ReactTimelineSeries";
+import { React, render, TextList } from "@hpcc-js/react";
+
+export class ReactAxisLabelTimeline extends Border2 {
+
+    private _listRenderer: React.FunctionComponent = TextList;
+    protected _gantt = new ReactTimelineSeries()
+        .columns(["Label", "start", "end", "idx"])
+        .mouseMode("marqueeSelection")
+        .zoomable(false)
+        .seriesColumn("idx")
+        ;
+
+    protected _listWrapper: HTMLWidget = new HTMLWidget();
+
+    constructor() {
+        super();
+        this.leftOverflowY("hidden");
+    }
+
+    enter(domNode, element) {
+        super.enter(domNode, element);
+
+        this._gantt.click = (row, col, sel) => {
+            this.click(row, col, sel);
+        };
+
+        this._gantt.dblclick = (row, col, sel) => {
+            this.dblclick(row, col, sel);
+        };
+
+        this.left(this._listWrapper);
+        this.center(this._gantt);
+
+        this.resizeWrappers();
+    }
+
+    update(domNode, element) {
+        super.update(domNode, element);
+        this.leftWidth(this.labelWidth());
+        const lhsNode = this._leftWA.element().node();
+
+        render(
+            this._listRenderer,
+            {
+                paddingTop: 22,
+                paddingBottom: 24,
+                rowHeight: this.bucketHeight() + this.gutter(),
+                fontFamily: this.labelFontFamily(),
+                fontSize: this.labelFontSize(),
+                data: this._gantt.data().map(n => n[0]),
+                onItemClick: (evt) => {
+                    console.log("evt", evt);
+                }
+            },
+            lhsNode
+        );
```

Comment:
This should be moved into a new Widget:  `class TextListWidget extends HTMLWidget`